docs/prepare-a-shared-rp-development-environment.md

```diff
@@ -418,20 +367,19 @@ az ad app credential reset \
    export AZURE_GATEWAY_CLIENT_ID='$AZURE_GATEWAY_CLIENT_ID'
    export AZURE_GATEWAY_SERVICE_PRINCIPAL_ID='$(az ad sp list --filter "appId eq '$AZURE_GATEWAY_CLIENT_ID'" --query '[].id' -o tsv)'
    export AZURE_GATEWAY_CLIENT_SECRET='$AZURE_GATEWAY_CLIENT_SECRET'
-   export RESOURCEGROUP="$RESOURCEGROUP_PREFIX-\$LOCATION"
-   export PROXY_HOSTNAME="vm0.$PROXY_DOMAIN_NAME_LABEL.\$LOCATION.cloudapp.azure.com"
-   export DATABASE_NAME="\$AZURE_PREFIX"
+   export RESOURCEGROUP='$RESOURCEGROUP_PREFIX-\$LOCATION'
+   export PROXY_HOSTNAME='vm0.$PROXY_DOMAIN_NAME_LABEL.\$LOCATION.cloudapp.azure.com'
+   export DATABASE_NAME='\$AZURE_PREFIX'
    export RP_MODE='development'
    export PULL_SECRET='$PULL_SECRET'
```

Comment:
I am pretty confident that USER_PULL_SECRET is missing here, but IDK how to create/find it.

Reply:
Does make secrets get you the pull secret?

Reply:
The pull secret mentioned here only consists for tokens for prod and int svc acr and not to say other registries like redhat or quay, these can provided to cluster during install or at later stages, RP doesn't need them and not a blocker for cluster creation.

Reply:
>Does make secrets get you the pull secret?

It does, but I am concerned about the **USER_PULL_SECRET** which is missing in the docs and exists in practice when you run `SECRET_SA_ACCOUNT_NAME=rharosecretsdev make secrets`.